Server/datalayer/time-entry.datalayer.js

```diff
@@ -1,12 +1,60 @@
+const timeentries = require('../models/time-entry.model');
 
 /*
  * @class timeEntriesDatalayer
  */
 class timeEntriesDatalayer {
 
     /**
-    * Initializes the datalayer
-    */
+     * Initializes the datalayer
+     */
+    static getTimeEntriesByUserId(obj) {
+
+        let param = {
+            "userId" : obj.userId,
+            "limit" : parseInt(obj.limit)
+        }
+        let sortOrder = {
+            '_id': -1
+        };
```

Comment:
Assume sortorder to be passed as well as keywords, e.g. `asc`, `desc`.
This one we can take it as default

Reply:
Updated